packages/editor/src/components/widgets/code-editor/ScriptArea.tsx

```diff
@@ -4,13 +4,30 @@ import ResizableCodeEditor from './ResizableCodeEditor';
 import { Browser, useBrowser } from '../../../components/browser';
 import { useMonacoEditor } from './useCodeEditor';
 import { usePath } from '../../../context';
+import MaximizedCodeEditorBrowser from '../../browser/MaximizedCodeEditorBrowser';
 
-const ScriptArea = (props: CodeEditorAreaProps) => {
+type ScriptAreaProps = CodeEditorAreaProps & {
+  openState: {
```

Comment:
maybe maximizeState or something like that is a bit clearer for a user of the ScriptArea...